log/logger_impl.go

```diff
@@ -89,12 +90,17 @@ func (l *logger) Fatal() Logger {
 }
 
 func (l *logger) Log(msg string) {
-	keyvals := make([]interface{}, (len(l.ctx)*2)+2)
+	orig := []string{
+		"msg", msg,
+		"time", time.Now().UTC().Format(time.RFC3339),
```

Comment:
I went with `"time"` here as the key. I like `ts` too but two extra characters avoid an acronym.

Reply:
I'd leave it `ts` as our system already looks at it without additional configuration.

Reply:
I don't think Loki cares but we have seen ts already populated.

Reply:
I'll change it to `ts`